src/components/section-contact/section-contact.scss

```diff
@@ -0,0 +1,68 @@
+$bg-color-blue: #114c7f;
+
+.section-contact {
+  background-color: #472f51;
+  .title {
+    margin: 0 auto;
+    max-width: 375px;
+    background-color: $bg-color-blue;
+    padding: 15px;
+    margin-top: -11px;
+    font-weight: 700;
+    h3 {
+      text-align: center;
+      font-size: 16px;
+      color: white;
+      margin: 0;
+    }
+  }
+  .info-box {
+    padding: 116px 0 0px;
+    text-align: center;
+    color: white;
+    p {
+      font-size: 21px;
+      font-weight: 300;
+    }
+  }
+  .form-box {
+    padding-top: 40px;
+    padding-bottom: 200px;
+    .form-box--name {
+      ::placeholder {
+        color: #62dafc;
+      }
+      .name-input {
+        border: 2px solid #62dafc;
+        border-radius: 0px;
+        background-color: transparent;
+        font-size: 16px;
+        font-weight: 700;
+        padding: 25px;
+        color: #62dafc;
+      }
+    }
+    .form-box--email {
+      ::placeholder {
+        color: #bd4573;
+      }
+      .email-input {
+        border: 2px solid #bd4573;
+        border-radius: 0px;
+        background-color: transparent;
+        font-size: 16px;
+        font-weight: 700;
+        padding: 25px;
+        color: #bd4573;
+      }
+    }
+    button {
```

Comment:
proponuje utworzyc button ktory bedzie mial zmienne tylko kolory 
aby wspolne style pogrupować:
https://getbootstrap.com/docs/4.1/components/buttons/
tak samo jak to robi bootstrap :)